memo.rb

```diff
@@ -1,25 +1,24 @@
 # frozen_string_literal: true
 
 require 'sinatra'
-require 'json'
+require 'pg'
 
+CONN = PG.connect(dbname: 'sinatra_memo_db')
 
-MEMOS_FILE = './memos.json'
-
-def load_memos
-  File.open(MEMOS_FILE) { |file| JSON.parse(file.read) }
+def load_memo
+  CONN.exec_params('SELECT * FROM memos WHERE id=$1', load_params.values_at(:id)).first
 end
 
-def save_memos(memos)
-  File.open(MEMOS_FILE, 'w') { |file| JSON.dump(memos, file) }
+def load_params
+  { title: params[:title], content: params[:content], id: params[:id] }
 end
 
 not_found do
   '存在しないページです'
 end
 
 get '/memos' do
-  @memos = load_memos
+  @memos = CONN.exec('SELECT * FROM memos ORDER BY id')
```

Comment:
```suggestion
  @memos = load_memos
```

みたいな感じで、load_memo と同じようにSQLを使う処理は全部別メソッドにすると実装の一貫性が出てなおかつgetやpostの中身がスッキリするので良いと思います